src/Bundle/Form/Extension/HttpFoundation/HttpFoundationRequestHandler.php

```diff
@@ -60,7 +60,7 @@ public function handleRequest(FormInterface $form, mixed $request = null): void
                     return;
                 }
 
-                $data = $request->query->get($name);
+                $data = $request->query->all()[$name];
```

Comment:
```suggestion
                $data = $request->query->all($name);
```

Cosmetic change 😅

Reply:
@mpysiak note for the future, please either apply suggestions from the comment or answer why you don't want to do so. Please, don't just mark a comment as resolved without anything in it

Reply:
@lchrusciel, it looks like this "cosmetic change" breaks in SF 5.4. Although it works in SF6. It produces an error
`Unexpected value for parameter "xxx": expecting "array", got "string"`

It works fine if I revert back to 
```php
$data = $request->query->all()[$name];
```